Revolver Rail light pixel fix

Hughgent opened the pull request:
## About The Pull Request

One pixel lower

## Why It's Good For The Game

aaaahhhhh
![scream1](https://user-images.githubusercontent.com/45076386/89964346-86a80580-dc0f-11ea-938d-9dd04d12c234.PNG)

AHHHHHH
![scream2](https://user-images.githubusercontent.com/45076386/89964366-8e67aa00-dc0f-11ea-9878-33656baeacb4.PNG)

**AAAAAAAHHHHHHHH**
![scream3](https://user-images.githubusercontent.com/45076386/89964388-97f11200-dc0f-11ea-9e7f-f2e6a7dc4ad4.PNG)

phew
![ahhh](https://user-images.githubusercontent.com/45076386/89964397-9d4e5c80-dc0f-11ea-9205-655c29e2f093.PNG)


## Changelog
:cl: Hughgent
fix: Standard Revolver rail attachments no longer hover.
/:cl:

<!-- Both :cl:'s are required for the changelog to work! You can put your name to the right of the first :cl: if you want to overwrite your GitHub username as author ingame. -->
<!-- You can use multiple of the same prefix (they're only used for the icon ingame) and delete the unneeded ones. Despite some of the tags, changelogs should generally represent how a player might be affected by the changes rather than a summary of the PR's contents. -->

psykzz commented:
I would've put it here (excuse the mspaint job)

![image](https://user-images.githubusercontent.com/1134201/89993548-c823c980-dc7e-11ea-8766-71588d42ec7b.png)

Hughgent commented:
Unfortunately, the variable I changed is for all the rail based attachments.

So if I moved it there, then we'd have side mounted red dots and such.

psykzz commented:
Can each attachment have its own offset, or is it just a mount point on gun?

Hughgent commented:
So it comes down to visualization of the 'slot'

Each gun has its own 'attachment point' the pixel x and y where that 'slot' is located.

Each attachment has it's own pixel x and y which adjusts how it looks when attached.

So we can adjust the weapons 'slot' but that adjusts all sprites that attach to that 'slot'. We can also adjust the attachment, but that adjusts the attachment for all weapons.

So in this case, the 'rail light' takes the 'rail slot' and we've decided that the 'rail slot' is on top of the weapon where most sights go.

From your picture, you'd probably want to make an 'under barrel light' instead. Which would be it's own thing.
